src/lib/Conversation.ts

```diff
@@ -114,7 +114,7 @@ export class Conversation {
     }
   }
 
-  async consentState(): Promise<"allowed" | "blocked" | "unknown"> {
+  async consentState(): Promise<"allowed" | "denied" | "unknown"> {
```

Comment:
i see this string union type defined twice (here and in `index.ts`). think it's worth it to create 1 type and reference that instead?

Reply:
Good call.